docs/DEV_WORKFLOW.md

```
@@ -0,0 +1,183 @@
+supabase db pull             # Downloads prod migrations
+npm run dev                  # Dev server uses local DB
+supabase db reset            # Between features: clean slate
+```
+
+---
+
+## Environment variables
+
+Required vars are validated on `npm run check:env`. All are stored in `.env.local`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)